### PR TITLE
fix: make "we are more than... green squares" slogan accessible

### LINKED ISSUE #88: Feature: Green squares is not accesible
### Type of feature

🍕 Feature

### Current behavior

reported [here](https://twitter.com/mislav/status/1603062302420668416). 

> The “We are more than green squares” graphic is visually cool, but is marked up by 4 distinct block elements. A screen reader would likely read that as 4 paragraphs. The graphic could be marked up by a single <p> containing spans to chop it up visually via CSS.

Screen Shot 2022-12-14 at 9 35 50 AM

### Suggested solution

This image should convert to a custom CSS.

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by CBID2**
## Description
This PR adds `<span>` elements to the "We Are More Then Green Squares" image, making it easier for screen reader users to navigate the website. 

## What type of PR is this? (check all applicable)

- [X] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents
Fixes #88 

## Mobile & Desktop Screenshots/Recordings
**Note: The audio does not play due to technical difficulties** 

https://github.com/open-sauced/landing-page/assets/105683440/c56fd2ba-e537-41a9-8e28-1cf9c9efb1f9

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [X] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [X] 🙅 no documentation needed

## Note to Reviewers
I used [this tool](https://bloggerpilot.com/en/tools/svg-to-css/) to get the CSS.

### CODE REVIEW

**Comment by nickytonline** [comment]
@CBID2, here's my suggestion. TLDR, add one paragraph element around the current paragraphs. Change the current paragraphs to spans, and at the end add a span with the text green squares with the `sr-only` class to make it only visible to screen readers. For the image, make the alt text empty (don't remove it).

That should make this more accessible.

```
<span className="sr-only">green squares</span>
```

```diff
-      <div className="flex pb-16 px-10 tablet:px-0 items-center justify-center largeTablet:pb-14">
-        <div>
-          <p className="text-textPrimary font-inter font-bold text-[2.375rem] leading-[3.625rem] largeTablet:text-5xl largeTablet:leading-[4.5rem]">
-            We are
-          </p>
-          <p className="font-inter font-bold pl-16 tablet:pl-20 text-[3.25rem] leading-none largeTablet:text-8xl">
-            <DecoratedText content="$orange-to-yellowmore" />
-          </p>
-          <p className="font-inter font-bold pl-32 tablet:pl-40 tracking-[-0.03em] text-[4.25rem] leading-none largeTablet:text-9xl">
-            <DecoratedText content="$orange-to-yellowthan..." />
-          </p>
-        </div>
-      </div>
-      <div className="pb-16 largeTablet:pb-52">
-        {isLargeTablet ? (
-          <Image alt="Green Squares text" src={GreenSquares} />
-        ) : (
-          <Image alt="Green Squares text" src={GreenSquaresSmall} />
-        )}
-      </div>
+       <div className="flex pb-16 px-10 tablet:px-0 items-center justify-center largeTablet:pb-14">
+         <p>
+           <span className="block text-textPrimary font-inter font-bold text-[2.375rem] leading-[3.+ 625rem] largeTablet:text-5xl largeTablet:leading-[4.5rem]">
+             We are
+           </span>
+           <span className="block font-inter font-bold pl-16 tablet:pl-20 text-[3.25rem] leading-none + largeTablet:text-8xl">
+             <DecoratedText content="$orange-to-yellowmore" />
+           </span>
+           <span className="block font-inter font-bold pl-32 tablet:pl-40 tracking-[-0.03em] text-[4.+ 25rem] leading-none largeTablet:text-9xl">
+             <DecoratedText content="$orange-to-yellowthan..." />
+           </span>
+           <span className="sr-only">green squares</span>
+         </p>
+       </div>
+       <div className="pb-16 largeTablet:pb-52">
+         {isLargeTablet ? (
+           <Image alt="" src={GreenSquares} />
+         ) : (
+           <Image alt="" src={GreenSquaresSmall} />
+         )}
+       </div>
```

**Comment by CBID2** [comment]
@nickytonline and @bdougie, I made the necessary changes but for some reason I keep getting errors. So far, I tried adding a variable called `largeTabletImage` but it did not work. Any other tip?

**Comment by BekahHW** [comment]
@CBID2 can you share the error message?

**Comment by nickytonline** [comment]
I'm going to go ahead and close this as it's been open since September 2023. Feel free to reopen if you plan to continue working on this.

**Comment by BekahHW** [comment]
Hey @CBID2 if the build is failing, it's not ready for review. Please go the through the process of checking the build before requesting a review. image

**Comment by CBID2** [comment]
> Hey @CBID2 if the build is failing, it's not ready for review. Please go the through the process of checking the build before requesting a review. image

Oh sorry @BekahHW.

File: components/sections/home-page/GitHubMock.tsx [hunk 1]
```diff
@@ -22,28 +22,30 @@ interface GitHubMockProps {
...
++           <span className="block font-inter font-bold pl-32 tablet:pl-40 tracking-[-0.03em] text-[4.+ 25rem] leading-none largeTablet:text-9xl">
++             <DecoratedText content="$orange-to-yellowthan..." />
++           </span>
++           <span className="sr-only">green squares</span>
++         </p>
++       </div>
++       <div className="pb-16 largeTablet:pb-52">
++         {isLargeTablet ? (
++           <Image alt="" src={GreenSquares} />
++          :
```
Comment (Arol15) [inline components/sections/home-page/GitHubMock.tsx:45]:
```suggestion
+         ) : (
```

File: components/sections/home-page/GitHubMock.tsx [hunk 2]
```diff
@@ -22,28 +22,30 @@ interface GitHubMockProps {
...
++           </span>
++           <span className="sr-only">green squares</span>
++         </p>
++       </div>
++       <div className="pb-16 largeTablet:pb-52">
++         {isLargeTablet ? (
++           <Image alt="" src={GreenSquares} />
++          : 
++           <Image alt="" src={GreenSquaresSmall} />
++         }
```
Comment (Arol15) [inline components/sections/home-page/GitHubMock.tsx:47]:
```suggestion
+         )}
```

File: components/sections/home-page/GitHubMock.tsx [hunk 3]
```diff
@@ -22,28 +22,30 @@ interface GitHubMockProps {
 
 const GitHubMock: FC<GitHubMockProps> = ({ moreHeading, topFeature }) => {
   const isLargeTablet = useMediaQuery()
+  const imageSource = isLargeTablet ? GreenSquares : GreenSquaresSmall;
```
Comment (Arol15) [inline components/sections/home-page/GitHubMock.tsx:25]: i don't see this being used anywhere:
```suggestion

```

File: components/sections/home-page/GitHubMock.tsx [hunk 4]
```diff
@@ -87,7 +88,5 @@ const GitHubMock: FC<GitHubMockProps> = ({ moreHeading, topFeature }) => {
         </div>
       </div>
     </SectionWrapper>
-  )
```
Comment (shiftyp) [inline components/sections/home-page/GitHubMock.tsx:90]: This parenthesis has been removed, this is probably the cause of your syntax error.

Comment (Arol15) [reply components/sections/home-page/GitHubMock.tsx:90]: nice catch @shiftyp! I missed it when I looked at it. @CBID2 you can leave these changes and apply the same in `components/sections/teams/GitHubMock.tsx`.

Comment (Arol15) [inline components/sections/home-page/GitHubMock.tsx:90]: @CBID2 you need to add this back in.

Comment (CBID2) [reply components/sections/home-page/GitHubMock.tsx:90]: I'm not near my computer yet @Arol15. Can you add the parentheses as a suggestion?

Comment (Arol15) [reply components/sections/home-page/GitHubMock.tsx:90]: Unfortunately, GitHub doesn't allow adding suggestions on deleted code.

Comment (CBID2) [reply components/sections/home-page/GitHubMock.tsx:90]: Ok I'll do it once I get on there

File: components/sections/teams/GitHubMock.tsx [hunk 5]
```diff
@@ -24,14 +24,18 @@ const GitHubMock: FC<GitHubMockProps> = ({ moreHeading, topFeature }) => {
     <SectionWrapper pbs={100}>
       <div className="flex pb-16 px-10 tablet:px-0 items-center justify-center largeTablet:pb-14">
         <div>
```
Comment (Arol15) [inline components/sections/teams/GitHubMock.tsx:26]:
```suggestion

```

File: components/sections/teams/GitHubMock.tsx [hunk 6]
```diff
@@ -24,14 +24,18 @@ const GitHubMock: FC<GitHubMockProps> = ({ moreHeading, topFeature }) => {
...
             <DecoratedText content="$orange-to-yellowmore" />
           </p>
-          <p className="font-inter font-bold pl-32 tablet:pl-40 tracking-[-0.03em] text-[4.25rem] leading-none largeTablet:text-9xl">
+          <span className="font-inter font-bold pl-32 tablet:pl-40 tracking-[-0.03em] text-[4.25rem] leading-none largeTablet:text-9xl">
             <DecoratedText content="$orange-to-yellowthan..." />
+            </span>
+            <span className="sr-only">green squares</span>
           </p>
         </div>
       </div>
```
Comment (Arol15) [inline components/sections/teams/GitHubMock.tsx:41]:
```suggestion

```

File: components/sections/teams/GitHubMock.tsx [hunk 7]
```diff
@@ -23,18 +23,20 @@ const GitHubMock: FC<GitHubMockProps> = ({ moreHeading, topFeature }) => {
   return (
     <SectionWrapper pbs={100}>
       <div className="flex pb-16 px-10 tablet:px-0 items-center justify-center largeTablet:pb-14">
-        <div>
-          <p className="text-textPrimary font-inter font-bold text-[2.375rem] leading-[3.625rem] largeTablet:text-5xl largeTablet:leading-[4.5rem]">
+          <p>
+          <span className="text-textPrimary font-inter font-bold text-[2.375rem] leading-[3.625rem] largeTablet:text-5xl largeTablet:leading-[4.5rem]">
             We are
+            </span>
           </p>
```
Comment (Arol15) [inline components/sections/teams/GitHubMock.tsx:30]:
```suggestion

```

File: components/sections/teams/GitHubMock.tsx [hunk 8]
```diff
@@ -23,18 +23,20 @@ const GitHubMock: FC<GitHubMockProps> = ({ moreHeading, topFeature }) => {
...
           <p className="font-inter font-bold pl-16 tablet:pl-20 text-[3.25rem] leading-none largeTablet:text-8xl">
```
Comment (Arol15) [inline components/sections/teams/GitHubMock.tsx:31]:
```suggestion
          <span className="font-inter font-bold pl-16 tablet:pl-20 text-[3.25rem] leading-none largeTablet:text-8xl">
```

File: components/sections/teams/GitHubMock.tsx [hunk 9]
```diff
@@ -23,18 +23,20 @@ const GitHubMock: FC<GitHubMockProps> = ({ moreHeading, topFeature }) => {
...
             <DecoratedText content="$orange-to-yellowmore" />
           </p>
```
Comment (Arol15) [inline components/sections/teams/GitHubMock.tsx:33]:
```suggestion
          </span>
```

**Comment by bdougie** [comment]
Though this page this exist, it has been moved. Closing this for now, since it has way to catch up.

**Comment by CBID2** [comment]
> Though this page this exist, it has been moved. Closing this for now, since it has way to catch up. 

Can I cherry pick the commits in a new PR for it @bdougie?

**Comment by bdougie** [comment]
That's fine

**Comment by CBID2** [comment]
> That's fine

Hey @bdougie. I made the new PR here: https://github.com/open-sauced/landing-page/pull/249

**Pull request closed without merging**